### PR TITLE
filter out include directories containing *intrin.h header files

### PR DESCRIPTION
**Pull request by gyorb**
There can be include directories with *intrin.h header files
not only at the implicit includes, this change
filters out any include directories containing *intrin.h header files.

### CODE REVIEW

**Comment by gyorb** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- analyzer/codechecker_analyzer/buildlog/log_parser.py  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/CodeChecker/codechecker/pullRequest?prid=4994075&bid=16483508)

File: analyzer/codechecker_analyzer/buildlog/log_parser.py [hunk 1]
```diff
@@ -1047,6 +1060,34 @@ def parse_options(compilation_db_entry,
             details['compiler_includes'][lang] = \
                 filter(__contains_no_intrinsic_headers, includes)
 
+        # filter out intrin directories
+        aop_without_itrin = []
```
Comment (csordasmarton) [inline analyzer/codechecker_analyzer/buildlog/log_parser.py]:
```suggestion
        aop_without_intrin = []
```

File: analyzer/tests/unit/test_option_parser.py [hunk 2]
```diff
@@ -487,6 +487,34 @@ def test_compiler_gcc_implicit_includes(self):
         self.assertTrue(any(map(lambda x: x.endswith('include-fixed'),
                                 res.compiler_includes['c++'])))
 
+    def test_compiler_intrin_headers(self):
+        """ Inlcude directories with *intrin.h files should be skiped."""
```
Comment (csordasmarton) [inline analyzer/tests/unit/test_option_parser.py]:
```suggestion
        """ Include directories with *intrin.h files should be skipped."""
```

**Comment by gyorb** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- analyzer/codechecker_analyzer/buildlog/log_parser.py  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/CodeChecker/codechecker/pullRequest?prid=4994075&bid=16483508)

**Comment by gyorb** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- analyzer/codechecker_analyzer/buildlog/log_parser.py  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/CodeChecker/codechecker/pullRequest?prid=4994075&bid=16483508)

**Pull request merged**